lib/events.js

```diff
@@ -21,16 +20,15 @@ const forwardEvent = (
   }
 };
 
-const forwardEvents = (
-  // Forward events from one EventEmitter to another
-  from, // EventEmitter, to listen for event
-  to, // EventEmitter, to emit event on
-  events // array of strings, events names
-  // Example: common.forwardEvent(from, to);
-  // Example: common.forwardEvent(from, to, 'eventName');
-  // Example: common.forwardEvent(from, to, { eventName: 'newEventName' });
-  // Example: common.forwardEvent(from, to, ['eventName1', 'eventName2']);
-) => {
+// Forward events from one EventEmitter to another
+//   from - EventEmitter, to listen for event
+//   to - EventEmitter, to emit event on
+//   events - array of strings, string or object, (optional), events names
+// Example: common.forwardEvent(from, to);
```

Comment:
This should be `forwardEvents`, look like a typo in an original code. We might as well fix it in this PR but in a separate commit.